changelog/v1.5.0-beta9/i3301-custom-ports.yaml

```
@@ -0,0 +1,4 @@
+changelog:
+    - type: NEW_FEATURE
```

fyi we have a `HELM` changelog type
```suggestion
    - type: HELM
```